_data/meltano/extractors/tap-googleads/matatika.yml

```diff
@@ -46,17 +46,32 @@ settings:
   kind: string
   label: Batch Config Storage Root
   name: batch_config.storage.root
-- description: Customer ID (Required) of the google ads account you want to sync data
-    from.
-  kind: password
+- description: |
+    Get data for customers by the provided IDs. Defaults to all customers accessible to
+    the authenticated principal.
+  label: Customer IDs
+  name: customer_ids
+- description: |
+    Get data for a customer by the provided ID. Defaults to all customers accessible to
+    the authenticated principal.
   label: Customer ID
   name: customer_id
```

Comment:
It seems that this is covered by `customer_ids` and it kept for backwards compatibility. Should we hide it?

Reply:
I've set it as hidden in 36342b8e4c122cf958a5416bcc5efd06f51e9409 - would you prefer a note in the description of `customer_id` and/or a separate PR to not render a section for hidden settings?

Reply:
(or I can just submit the Vue changes in this PR)

Reply:
Ah, I didn't realize the Hub doesn't do anything with the `hidden` flag. In that case it's easier to add a note that folks should use `customer_ids`.

The UI changes can be made in a separate PR (I also now realize it's part of https://github.com/meltano/hub/issues/1418).

Reply:
https://github.com/meltano/hub/pull/1872/commits/b81b3b396aa7844c9af29c265957b671aa9d1e4c